Sources/Host/POSIX/PTrace.cpp

```diff
@@ -125,6 +125,25 @@ ErrorCode PTrace::resume(ProcessThreadId const &ptid, ProcessInfo const &pinfo,
   return doStepResume(false, ptid, signal, address);
 }
 
+ErrorCode PTrace::getClonedTID(ProcessId pid, unsigned long &clonedTID) {
+#if defined(OS_LINUX)
+  auto res = wrapPtrace(PTRACE_GETEVENTMSG, pid, nullptr, &clonedTID);
+
+  if (res < 0) {
+    return Platform::TranslateError();
+  }
+
+  return kSuccess;
+#elif defined(OS_FREEBSD) || defined(OS_DARWIN)
+  // PTRACE_GETEVENTMSG does not exist on Darwin. TODO: Find alternative
+  // implementation. Alternatively, this method could be in Linux::PTrace, but
+  // various bugs exist
+  //   that cause Linux::Process to depend on it's Linux::Process::ptrace()
+  //   method returning a POSIX::PTrace and NOT a Linux::PTrace
```

Comment:
I'm not sure what the problem is here, you can move this function to `Host::Linux::PTrace`, and make `Target::Linux::PTrace` return a `Host::Linux::PTrace` instead of a `Host::POSIX::PTrace`. See PR#635.

Reply:
Yup, confused the purpose with some subset of flakes with this change.

Reply:
The whitespace is also strange, looks like some lines are double-indented?